src/mod_muc_room.erl

```diff
@@ -3295,13 +3295,12 @@ change_config(Config, StateData) ->
     case {(StateData#state.config)#config.persistent,
 	  Config#config.persistent}
 	of
-      {_, true} ->
-	  mod_muc:store_room(NSD#state.server_host,
-			     NSD#state.host, NSD#state.room, make_opts(NSD));
-      {true, false} ->
-	  mod_muc:forget_room(NSD#state.server_host,
-			      NSD#state.host, NSD#state.room);
-      {false, false} -> ok
+      {_, _} ->
```

Comment:
is it really any matcher? If thats the case feels like a case statement is redundant

Reply:
I have 2 specific cases where I want things to happen.  This makes it explicit.